test_cli.py

```diff
@@ -924,7 +975,9 @@ def test(ctx):
     # Modify to non-ascii tags
     # TODO: enable with Unicode support.
```

Comment:
Is this comment out of date?

Reply:
I'm not sure. It appears that we still run `validate_ascii` on tag names (https://github.com/codalab/codalab-worksheets/blob/1c2b326cf42e42f5b98657050cddcef69ca357cb/codalab/rest/schemas.py#L118) so I don't recall us ever adding support for unicode tags. Need to investigate this further, but that can probably be done separate from this PR.